chapters/fundamentals.txt

```diff
@@ -1256,19 +1256,22 @@ the application.
 ename:VK_ERROR_UNKNOWN will be returned by an implementation when an
 unexpected error occurs that cannot be attributed to valid behavior of the
 application and implementation.
+Under these conditions, it may: be returned from any command returning a
+elink:VkResult.
 
 [NOTE]
 .Note
 ====
-If ename:VK_ERROR_UNKNOWN is received, the application should be checked
-against the latest validation layers to verify correct behavior as much as
-possible.
+If ename:VK_ERROR_UNKNOWN is received, the application would benefit from
```

Comment:
The phrase "the application would benefit from being checked..." is quite awkward. If you don't like "the application should be checked", perhaps something like "recommended practice is to check the application..."? 

To be honest, this Note was already a mess. An alternative to this fix would be to reorder the statements like this:
> ename:VK_ERROR_UNKNOWN is not expected to be returned from any command if application behavior is valid and the implementation is bug-free. If the error is received, recommended practice is to use the latest validation layers to check for invalid behavior on the part of the application. If no invalid usage is detected, report the error to the implementor as a possible implementation bug.

Reply:
I mostly want to deal with the RFC keywords. Did not really wanted to get suck rewording this Note.
If I just replace "should" with "recommended" I am back to square one. I guess I will just revert this for the time being.